src/main/java/frc/robot/commands/IntakeCommand.java

```diff
@@ -9,23 +12,35 @@ public class IntakeCommand extends Command {
   private Intake intakeInstance;
   private Kicker kickerInstance;
   private Shooter shooterInstance;
+  private ArmPID armInstance;
 
   /** Creates a new IntakeCommand. */
   public IntakeCommand() {
     // Use addRequirements() here to declare subsystem dependencies.
     intakeInstance = Intake.getInstance();
     kickerInstance = Kicker.getInstance();
     shooterInstance = Shooter.getInstance();
+    armInstance = ArmPID.getInstance();
+    // arm is NOT a requirment as it is not being moved, just checking the angle
     addRequirements(intakeInstance, kickerInstance, shooterInstance);
   }
 
   // Called when the command is initially scheduled.
   @Override
   public void initialize() {
     // starts motors
-    intakeInstance.startIntaking();
-    kickerInstance.startIntaking();
-    shooterInstance.setReverse();
+    if (armInstance.getMeasurement() <= (Constants.ArmConstants.zeroPosition
+        + Units.degreesToRadians(Constants.IntakeConstants.armPosMarginError))) {
+      intakeInstance.startIntaking();
+      kickerInstance.startIntaking();
+      shooterInstance.setReverse();
+    }
+    /*
```

Comment:
No need to leave these lines as comments, just remove them as you just wrapped them in an if statement.  In general you shouldn't comment out code you are changing just delete it as it's still viewable in previous commits.